tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -288,12 +288,12 @@ install_kernel_nvidia_gpu() {
 
 #Install GPU and SNP enabled kernel asset
 install_kernel_nvidia_gpu_snp() {
```

Comment:
Worth renaming the function as well? `install_kernel_nvidia_gpu_sev`

Reply:
Technically we're enabling SNP not only SEV. The resulting kernel is named SNP so I want to keep this. It may confuse people if they see "only" SEV when doing `uname -a`.